2020-an-environment-for-sustainable-research-software-in-germany-and-beyond/manuscript.md

```diff
@@ -0,0 +1,2021 @@
+impactful results. Not all researchers thinking of themselves as RSEs
+pursue a faculty position as their main career goal. However, permanent
+non-professorial academic faculty positions are rare within the German
+academic system, also due to the lack of a defined **RSE role**. We
+recommend research institutions to leverage the benefit of dedicated
+RSEs by establishing attractive long-term career options in the academic
+environment. The long-term solution in order to gain sufficient software
+development **skills** should be education that is included early in the
+career path, ideally already at the Bachelor level. For the time being
+however, efforts involving workshops and seminars that provide easy
+access to hands-on training on software-related questions should be
+promoted and supported as much as possible.
+
+It is important to provide an environment where **communities** can form
+and flourish by allocating resources for research software development
+and for building communities around it (Katz et al. 2019; Iaffaldano et
+al. 2019; Jiménez et al. 2017). The identification with a community of
+like-minded people and personal action (Allen et al. 2017) can lead to a
+permanent establishment of sustainable research software as a valuable
+research output. Thus, research institutions as well as funding agencies
+should not only be open-minded regarding existing volunteer
+organizations, but should actively promote the creation of such groups.
+
+How can Research Software be Sustainably Funded?
+================================================
+
+> *Hal9k* has grown into a widely used software in researchonomics, and
+> Kim is proactively asked to apply for - and is subsequently awarded -
+> a permanent RSE position at the institute for researchonomy at UofA,
+> based on her work on the library. She works closely with the central
+> RSE team, but mostly due to bureaucracy and the high demand for her
+> library, Kim does not have enough time to maintain and further develop
+> *hal9k* alone anymore. Together with the dean she develops a course
+> for the researchonomics curriculum which teaches data processing with
+> *hal9k*. As a lesson from her own career, she starts the course with
+> sessions on the Hash shell, version control with Tig, Boa, and two
+> whole sessions on basics of sustainable software development. This is
+> very fruitful, and due to the implementation of a new research
+> software funding scheme at UofA, Kim is able to hire one of the course
+> students, who has shown great research software engineering skills
+> straight into a long-term position at her institute, where they focus
+> on the maintenance and development of *hal9k*, work with the computing
+> center to support *hal9k*-based supercomputing on a new, dedicated
+> FGPA cluster, develop training materials for external users, and
+> organize the yearly *hal9k* users and developers conference. Kim gets
```

Comment:
As far as I understood, there are only 2 developer of *hal9k* atm (Kim and the student she hired). Should we mention others in this part? Other researcher / RSEs who extend the software because they need the functionality?

Reply:
Should we also mention community management beyond events? In my experience this is a very important part. See also "The Art of Community: Building the New Age of Participation, Jono Bacon" https://www.amazon.de/gp/product/B008224FMC/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1

Reply:
I would prefer not to make the story more complex, but rather have the community management included down below in line 997

Reply:
> As far as I understood, there are only 2 developer of _hal9k_ atm (Kim and the student she hired). Should we mention others in this part? Other researcher / RSEs who extend the software because they need the functionality?

At this stage, there have been collaborations with other research facilities, so I guess rather few developers & quite a few users. Good enough to run a conference for them I think?